DEV-GUIDE.md

```diff
@@ -0,0 +1,78 @@
+# SPIRIT DS GUIDE FOR DEVELOPER
+
+## Documentation
+
+Before the actual implementation, it is advisable to prepare a file with documentation, where individual
+properties of the component and parameters will be described.
+
+## Tests
+
+Before test must be built icon package, run `% yarn build` in `packages/icons/package.json`.
+In local development you need to run test in each package.
+
+### Web
+
+- `% cd <your-local-path>/spirit-design-system/packages/web`
+- `% yarn test:unit`
+
+### Web React
+
+- `% cd <your-local-path>/spirit-design-system/packages/web-react`
+- `% yarn test:unit`
+
+## Commit
+
+### Run before commit
+
+1. Prettier `% yarn format:fix`
+2. Types `% yarn types`
+3. Lint `% yarn lint`
+4. Test in each package
+
+### Commit message
+
+Message of commit must contain specific phrases, like `<type>(<package>): <commit-message> #<issue-id>` eg.`Feat(web-react): React Modal implementation #DS-331`
+
+## Development
+
+Each package has different development steps, so:
```

Comment:
This should be maybe an issue? What do you think? We want to start every package the same way and that should be `yarn start` or `yarn dev`.

For now I think that this information should be in README of every package.